WORKSPACE

```diff
@@ -60,22 +60,23 @@ api_dependencies()
 # Following go repositories are for building go integration test for mixer filter.
 git_repository(
     name = "io_bazel_rules_go",
-    commit = "7991b6353e468ba5e8403af382241d9ce031e571",  # Aug 1, 2017 (gazelle fixes)
+    commit = "9cf23e2aab101f86e4f51d8c5e0f14c012c2161c",  # Oct 12, 2017 (Add `build_external` option to `go_repository`)
     remote = "https://github.com/bazelbuild/rules_go.git",
 )
 
-git_repository(
-    name = "org_pubref_rules_protobuf",
-    commit = "9ede1dbc38f0b89ae6cd8e206a22dd93cc1d5637",
-    remote = "https://github.com/pubref/rules_protobuf",
-)
+load("@io_bazel_rules_go//go:def.bzl", "go_rules_dependencies", "go_register_toolchains")
+go_rules_dependencies()
+go_register_toolchains()
+
+load("@io_bazel_rules_go//proto:def.bzl", "proto_register_toolchains")
+proto_register_toolchains()
 
-MIXER = "535eb564667cef6aed334cb4f5e967a104768387"
+MIXER = "dadb025561a8c11720e6d6ccf54bbdbb29c6bf96"
 
 git_repository(
     name = "com_github_istio_mixer",
     commit = MIXER,
-    remote = "https://github.com/istio/mixer",
+    remote = "https://github.com/lizan/mixer",
```

Comment:
why is from your fork?

Reply:
Because it is WIP, need to merge underlying PR before merge this.